custom_models/mental_health/models/mental_health_notes.py

```diff
@@ -11,3 +11,89 @@ class MentalHealthNotes(models.Model):
     icd10_codes = fields.Selection(string='ICD10 Code', required=False, copy=False,
                                    selection=[('f40.00', 'F40.00 - Agoraphobia unspecified'),
                                               ('f40.01', 'F40.01 - Agoraphobia with panic disorder')])
+    thought_process = fields.Selection(string='Thought Process', copy=False,
```

Comment:
Should this field be required if a "normal" thought process is an option?

Reply:
Same goes for most of these fields tbh

Reply:
I agree about making these fields required, and I think a number of them should also default to the neutral choice. That will be convenient for the user. @DiamondDagger590

Reply:
I agree, it all looks good from a QA standpoint, but I think we should get the default values set before it gets merged.

Reply:
I set the default values and made the fields required. QA anybody?